pysph/tools/ipy_viewer.py

```diff
@@ -683,9 +721,9 @@ def __init__(self, particlearray):
         self.scalar_cmap.owner = self.array_name
         self.velocity_vectors = widgets.Checkbox(
             value=False,
-            description="Vectors",
+            description="Velocity Vectors",
             disabled=False,
-            layout=widgets.Layout(width='100px', display='flex')
+            layout=widgets.Layout(width='300px', display='flex')
```

Comment:
Do we need fixed sizes here?  Does it render OK on a smaller window?

Reply:
This is just to make sure that the label 'velocity vectors' is displayed completely. It renders fine on a smaller screen, ipywidgets just creates a scroll bar to traverse all the particle arrays.